data-service/src/test/java/com/amazonaws/blox/dataservice/integration/DescribeEnvironmentIntegrationTest.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright 2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"). You may
+ * not use this file except in compliance with the License. A copy of the
+ * License is located at
+ *
+ *     http://aws.amazon.com/apache2.0/
+ *
+ * or in the "LICENSE" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+package com.amazonaws.blox.dataservice.integration;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import com.amazonaws.blox.dataservicemodel.v1.model.EnvironmentId;
+import com.amazonaws.blox.dataservicemodel.v1.model.wrappers.DescribeEnvironmentResponse;
+import org.junit.Test;
+
+public class DescribeEnvironmentIntegrationTest extends DataServiceIntegrationTestBase {
+  private static final String ACCOUNT_ID = "123456789012";
```

Comment:
Looks the "ACCOUNT_ID", "ENVIRONMENT_NAME" and `envionrmentId` are pretty much the same for all the test,  we most likely could move into the base class.   Even some of the test case need different environment or cluster, we can have different name for those.

Reply:
The ENVIRONMENT_NAME and CLUSTER can be different, but I can use the customized value for them